### PR TITLE
Dockerize basic django set-up

### LINKED ISSUE #4: Dockerize Django run
Create a simple Docker wrapper for running Django runserver so that the website can be deployed from a container instead of a virtual environment.

### PR DESCRIPTION
**Pull request by dchiller**
This PR introduces a basic dockerized setup of a django + postgres backend for VIM. The README is updated with basic instructions for local development. NOTE: As mentioned in the README, this should not be deployed to any remote server.

Closes #4. Addresses some of #2.

### CODE REVIEW

File: web-app/django-startup.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+python manage.py makemigrations
+python manage.py migrate
```
Comment (jacobdgm) [inline web-app/django-startup.sh:4]: my understanding is that it's best practice to include migrations in the git history. Are we confident there will never be a situation where a developer wants to start the server without making new migrations and/or migrating?

**Review by jacobdgm — APPROVED** [review]
It's not clear to me why we need `django-startup.sh` or the code contained therein, but on the whole, this looks good. Approving.

File: web-app/django-startup.sh (hunk 1 above)
Comment (dchiller) [reply web-app/django-startup.sh:4]: good point!

**Comment by dchiller** [comment]
This "dismiss the stale review" happened automatically .... oops ... just changed that option

**Review by jacobdgm — APPROVED** [review]

**Review by lucasmarchd01 — APPROVED** [review]

**Pull request merged**